cinema/serializers.py

```diff
@@ -1,6 +1,8 @@
 from rest_framework import serializers
 
-from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+from cinema.models import (Genre, Actor,
+                           CinemaHall, Movie,
+                           MovieSession, Order, Ticket)
```

Comment:
If importing more than two items, it is better to write them on separate lines. This improves readability and maintenance of the code. 

Example:

From taxi.models import ( 
	Driver,
	Manufacturer,
	Car
)
Read more about topic [here](https://peps.python.org/pep-0008/#imports)